nglview/utils/widget_utils.py

```diff
@@ -90,3 +92,66 @@ def func(this, **kwargs):
             func = make_func(rep)
             fn = '_'.join((root_fn, rep[0]))
             setattr(self, fn, MethodType(func, other))
+
+
+def compare_two_images(image_path_1,
+                       image_path_2,
+                       standard_width=200,
+                       standard_height=200,
+                       convert_to_gray=False,
+                       threshold=0.97):
+    """
+    Comparing two images. Two images will be convert to new size
+    (standard_width, standard_height) and then compute structural similarity
+    to compare them.
+    Input:
+        image_path_1: string, path for the first image
+        image_path_2: string, path for the second image
+        standard_width: float (number of pixels) for a standard width
+        standard_height: float (number of pixels) for a standard height
+        convert_to_gray: boolean; if user want to convert a gray scale after
+            resize the images.
+        threshold: float, threshold for the structural similarity index
+            between two images. The default value is 0.97
+    Output:
+        Boolean value
+    """
+    from skimage.measure import compare_ssim as ssim
+
+    # load image and resize
+    img_1 = resize_and_load_image(image_path=image_path_1,
+                                  new_width=standard_width,
+                                  new_height=standard_height,
+                                  convert_to_gray=convert_to_gray)
+    img_2 = resize_and_load_image(image_path=image_path_2,
+                                  new_width=standard_width,
+                                  new_height=standard_height,
+                                  convert_to_gray=convert_to_gray)
+    b_multichanel = not convert_to_gray
+    s = ssim(img_1, img_2, multichannel=b_multichanel)
+
+    return s > threshold
+
+
+def resize_and_load_image(image_path,
```

Comment:
should this be private?

Reply:
may be just bring the part

```
    img_resize = cv2.resize(img, (new_width, new_height),
                            interpolation=cv2.INTER_AREA)
``` to `compare_two_images` and remove this `resize_and_load_image` method.